### DISCUSSION TITLE
Tracking multiple Hunt-Accounts on the same PC

### DISCUSSION THREAD

**Discussion by MadHoek** [majikat768/HuntStatsLogger#74] [Ideas]
I recently got a second copy of Hunt on a different Steam account.
(mainly to enjoy Soul Survivor without ruining my MMR as it is not sepaerated from Bounty Hunt)
I play both accounts on the same PC, so I only need to install Hunt once. As this obviously confuses the HuntStatsLogger because of a different username used in the attributes.xml when I switch to my alt-account, I searched for solutions and found something very nice.

The attributes.xml is stored in the Hunt installation directory in the following folder (as you already know):
...user\profiles\default\attributes.xml

1: I made a copy of the "default" folder and renamed it to my alt-steam-accounts-name
2: I opened "profiles.xml" found in the "alt-steam-accounts-name" folder and changed: Profile Name="default" to: Profile Name="alt-steam-accounts-name"
3: Now Hunt uses the "default" profile for my main-account and the "alt-steam-accounts-name" profile for my new account

That way one can have completely seperated profiles for different steam accounts if they both own a copy of the game.
Maybe you want to add the feature to track/switch different Hunt-accounts.

Btw, I watch the progress of your app since the early stages ....it improved a lot and I love it ...great work! Thanks a lot, for your effort.
Cheers mate

Edit:
Unfortunatelly, this is not how it works. Although one can have multiple profiles, it seems like they are taken by the game in alphabethical order (if there is another folder than the "default" present it seems to use this one instead)
Before I wrote this discussion I tested this the other day and I could have sworn it worked like I described it.
But today Hunt used my "alt-steam-accounts-name" profile as the default folder for each of my accounts.
Maybe there is a way to make Hunt use dedicated profile folders for different accounts, but for now I don't know how.
If I find out, I'll let you know.

Edit2:
As the "profileid" and the "blood_line_name" of both of my accounts are stored in the "attributes.xml" it may be possible to track both of them even without the use of different profile folders.

**Comment by majikat768** [comment]
Interesting, thanks!
yeah, I think the easiest way may be to implement a database file backup/restore system, so when you login to a different steam account you just switch over to another huntstats.db file along with setting the username...it'd be nice if it could be automatic though.  I think that could be doable with the profileid, as long as the two accounts are never in a match together (which, if they're alt accounts for one person, shouldn't happen).

**Comment by MadHoek** [comment]
Oh yea ...sometimes I think just too complicated.
Should work if I write a simple bat-file which replaces the "huntstats.db" & "settings .ini" along with the "attributes.xml" each time I switch the account.
It would be really nice if you could manage to integrate the automatic switch like you stated ...but I'm totally fine with the "bat-solution".
In case someone is interested how I do this, I described it here: https://github.com/majikat768/HuntStatsLogger/issues/53
Thanks for the response ...and your great app ...I can't thank you enough for it, it's that great :)
